templates/shortcodes/youtube.html

```diff
@@ -0,0 +1,3 @@
+<div {% if class %}class="{{class}}"{% endif %}>
+    <iframe src="https://www.youtube-nocookie.com/embed/{{id}}{% if playlist %}?list={{playlist}}{% endif %}{% if autoplay %}?autoplay=1{% endif %}" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
```

Comment:
Quick question. Do we have X-Frame-Options already set in our webserver to avoid possible https://developer.mozilla.org/en-US/docs/Web/Security/Types_of_attacks#click-jacking

Reply:
Looks like GitHub Pages doesn't set the `X-Frame-Options` header according to devtools.

My understanding is that the `X-Frame-Options` header issue prevents the site itself from being rendered in iframes, not iframes on the site, as in the shortcode here. 

Since it's not relevant on this PR, it would be good to create an issue so we can track it otherwise. There are [JS-level mitigations](https://supergeekery.com/blog/deter-clickjacking-when-you-cant-access-headers-iframe-javascript) that we might want to look into.